rock_paper_scissors/user.py

```diff
@@ -7,19 +7,18 @@ def __init__(self, name):
         self._choice = None
         self.name = name
 
+    def __repr__(self):
+        return self.name
```

Comment:
I like this, but recommend something a little more verbose, as well as enclosing the returned string in `<>` characters, as it's fairly standard practice to do so. It's also a good idea to include the name of the class in any `__repr__` string.

Something that produces output like this should work well:
`<User: human>`